### PR TITLE
Header, SiteNav, logos, icons and Next Step Backend Changes

### PR DESCRIPTION
**Pull request by dcloud**
**Description of change**

Created a SiteNav that lives on the left of the page, per the design. Moved navigation elements from the header to the SiteNav. The header is now just the logo and website title. 

I made the SiteNav and Header match the design as closely as possible, but the links near the bottom of the SiteNav are a lighter than the design because the gray didn't pass accessibility requirements around color contrast.

After discussing with Aricka and the team, we decided the SIteNav should take up the full viewport height and be pinned to the left side, so layout adjustments were made to accommodate that.

I also added some classes to `<body>` since it seemed we want some base font/text-color styles as the default. I had noticed some elements were rendering with system fonts rather than those from the DS.

Since I was working on the logo, I also created icons for the app manifest and favicons, and updated the meta description to match the title of the application.

*Backend Changes*
Backend changes were introduced to support adding notes from the Next Step page.
These notes relate to specialists and grantees. This does not include any Frontend changes, those will be introduced in another PR.

**How to test**

**Issue(s)**
* closes https://github.com/HHS/Head-Start-TTADP/issues/86
* closes https://github.com/HHS/Head-Start-TTADP/issues/93
* partial work on https://github.com/HHS/Head-Start-TTADP/issues/100 (backend only)

See also https://github.com/adhocteam/Head-Start-TTADP/pull/177

### CODE REVIEW

File: frontend/public/manifest.json [hunk 1]
```diff
@@ -3,9 +3,9 @@
   "name": "TTA Smart Hub",
   "icons": [
     {
-      "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/x-icon"
+      "src": "favicon.png",
```
Comment (rahearn) [inline frontend/public/manifest.json:6]: I don't see a `favicon.png` anywhere, should this be one of the logo.png files?

File: frontend/src/components/SiteNav.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            <div role="complementary" className="padding-2 smart-hub-sitenav-word-wrap--break">
+              <p className="text-bold margin-y-105">{ user.name }</p>
+              <p className="font-sans-3xs margin-y-105">{ user.email }</p>
+            </div>
+          </div>
+          <nav>
+            <div className="width-full margin-bottom-2">
+              <ul className="add-list-reset">
+                <li>
+                  <Link
```
Comment (rahearn) [inline frontend/src/components/SiteNav.js:63]: This list feels like it'll be more likely to be added to than the admin nav items, so I think the new `NavLink` helper function should probably reflect this use case, and not the one admin link.

**Review by rahearn — CHANGES_REQUESTED** [review]
Looks great. Hitting the logout button in the sidenav displays the "logged out due to inactivity" message though. Please fix that, and add (or fix) a frontend test that should have caught it.

Screen Shot 2021-02-24 at 9 51 43 AM

**Comment by dcloud** [comment]
Fixes are in https://github.com/adhocteam/Head-Start-TTADP/pull/192

**Review by rahearn — COMMENTED** [review]
This will be good to go once the description is edited to reflect the addition of https://github.com/adhocteam/Head-Start-TTADP/pull/173

**Review by rahearn — APPROVED** [review]

**Pull request merged**